third-party/ios/ExecutorchLib/ExecutorchLib/Exported/ETModel.mm

```diff
@@ -29,6 +29,105 @@ - (NSNumber *)loadForward {
   return @((int)err);
 }
 
+- (NSNumber *)getNumberOfInputs {
+  const auto method_meta = _model->method_meta("forward");
+  
+  if (method_meta.ok()) {
+    return @(method_meta->num_inputs());
+  }
+  
+  return @-1;
```

Comment:
`method_meta.error()` should be returned instead of a random -1. Actually, an error should be thrown with a message from `method_meta.error()`. Then that error should be caught within the ETModule.mm, and the promise should be rejected with a proper message.